.github/workflows/_buildx.yml

```diff
@@ -0,0 +1,45 @@
+name: 'Build Docker'
+
+on: workflow_call
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    name: Build Docker
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/download-artifact@v2
+        with:
+          name: dist
+      - name: Prep image files
+        run: |
+          mkdir -p {etc,binaries}
+          printf "root:x:0:0::/root:/bin/ash\nshiori:x:1000:1000::/shiori:/bin/ash" > etc/passwd
+          printf "root:x:0:root\nshiori:x:1000:shiori" > etc/group
+      - name: Prep binaries
+        working-directory: binaries
+        run: |
+          cp -r ../shiori_linux_* .
+          mv shiori_linux_arm_7 shiori_linux_arm
+          gzip -d -S .gz__  -r .
+          chmod 755 shiori_linux_*/shiori
+          ls -lRh
+      - name: Buildx
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login -u go-shiori --password-stdin ghcr.io
+          REPO=ghcr.io/go-shiori/shiori
+          TAG=$(git describe --tags)
+          if [ -z "$(git tag --points-at HEAD)" ]
+          then
+            TAG2="dev"
+          else
+            TAG2="latest"
+          fi
+          docker buildx create --use --name builder
+          docker buildx ls
+          printf 'FROM ghcr.io/ghcri/alpine:3.15\nARG TARGETARCH\nCOPY etc /etc\nCOPY binaries/shiori_linux_$TARGETARCH/shiori /usr/bin/shiori' > Containerfile
```

Comment:
Maybe we could have this `Containerfile` in the repo?

Reply:
The files that were generated in CI are now in ` .github/workflows/docker`.  The `Containerfile` is now  `Dockerfile.ci`. I also updated the main `Dockerfile` to more closely match `Dockerfile.ci`. The workflow is using pregenerated `etc/passwd` and `etc/group` files to prevent the use of qemu when running buildx.